### DISCUSSION TITLE
Add support for Multi Factor authentication in Firebase Auth

### DISCUSSION THREAD

**Discussion by cstranex** [invertase/react-native-firebase#4289] [Ideas] [answered]
Added in March to Firebase Authentication and Google Identity Platform, adds the ability to register and sign in users using multi-factor authentication methods (Such as SMS, Email).

See the docs for more info: https://cloud.google.com/identity-platform/docs/android/mfa

**Comment by mikehardy** [comment, marked as answer]
Duplicate: #4166 feature request there

It is important to say - there is no "feature fairy" in this module. This module needs people proposing PRs vs requesting features or they won't happen. I am personally willing to collaborate with contributors on PRs to get them released - and I do this very quickly! - but we all rely on each other for the features and we all move the module forward together.

To be very concise and clear: without a PR from the community, this will not happen, but I would love to see a PR!

**Comment by mikehardy** [comment]
MFA with phone support released as v11.3.0
